src/interface.jl

```diff
@@ -530,5 +614,30 @@ function ManifoldsBase.log(G::LieGroup, e::Identity, g)
     return X
 end
 
+# explicit method error to avoid stack overflow
 @doc "$(_doc_log_id)"
-ManifoldsBase.log!(::LieGroup, X, ::Identity, g)
+function ManifoldsBase.log!(G::LieGroup, X, e::Identity, g)
+    throw(MethodError(ManifoldsBase.log!, (typeof(G), typeof(X), typeof(e), typeof(g))))
+end
+
+LinearAlgebra.norm(G::LieGroup, g, X) = LinearAlgebra.norm(G.manifold, g, X)
```

Comment:
```suggestion
LinearAlgebra.norm(G::LieGroup, g, X) = norm(G.manifold, g, X)
```
We don't need to specify module for exported functions like `norm` when calling them.

Reply:
Ah, right, if we export them we do not, otherwise do. I sometimes di not fully have that in mind.